lib/cloudregister/smt.py

```diff
@@ -140,33 +138,35 @@ def is_equivalent(self, smt_server):
     def is_responsive(self):
         """Check if the SMT server is responsive"""
         # Per rfc3986 IPv6 addresses in a URI are enclosed in []
-        if self.get_ipv6():
-            health_url = 'https://[%s]/api/health/status' % self.get_ipv6()
-            cert_url = '%s://[%s]/smt.crt' % (self._protocol, self.get_ipv6())
-        else:
-            health_url = 'https://%s/api/health/status' % self.get_ipv4()
-            cert_url = '%s://%s/smt.crt' % (self._protocol, self.get_ipv4())
-
-        # We cannot know if the server cert has been imported into the
-        # system cert hierarchy, nor do we know if the hostname is resolvable
-        # or if the IP address is built into the cert. Since we only want
-        # to know if the system is responsive we ignore cert validation
-        # Using the IP address protects us from hostname spoofing
-        try:
-            response = requests.get(health_url, timeout=2, verify=False)
-            if response.status_code == 200:
-                status = response.json()
-                return status.get('state') == 'online'
-            elif response.status_code == 404:
-                # We are pointing to an SMT server, the health status API
-                # is not available. Download the cert to at least make sure
-                # Apache is responsive
-                cert_response = requests.get(cert_url, verify=False)
-                if cert_response and cert_response.status_code == 200:
-                    return True
-        except Exception:
-            # Something is wrong with the server
-            pass
+        ips = [self.get_ipv6(), self.get_ipv4]
```

Comment:
You're passing in the method `self.get_ipv4` instead of the result.

Reply:
Yes, I noticed it and corrected it already